src/player.cpp

```diff
@@ -50,20 +52,17 @@ double player::get_x() const noexcept { return m_c.get_x(); }
 /// Get the Y coordinate of the player
 double player::get_y() const noexcept { return m_c.get_y(); }
 
-/// Get the radius of the player
-double player::get_diameter() const noexcept { return m_diameter; }
-
 /// Make the player grow
 void player::grow()
 {
-  m_diameter *= m_growth_factor;
+  m_health *= m_growth_factor;
 }
 
 /// Make the player shrink
 void player::shrink()
 {
-  m_diameter /= m_growth_factor;
-  if(m_diameter <= m_death_size)
+  m_health /= m_growth_factor;
+  if(this->get_diameter() <= m_death_size)
```

Comment:
Remove completely diameter, use only health

Reply:
`player::get_diameter` converts the health to the diameter, that is why it is called. You suggest a free (as in: non-class non-member) function?